### PR TITLE
Adds upload delivery-reports command

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds the CLI command to upload several delivery-reports at once

How to test:
0. install on stage 
1. run add-delivery_report_created_at-to-analysis.sql
1. us
1. cg upload delivery-reports

Expected outcome:
The command should create a delivery-report for each analysis that doesn't already have one

### CODE REVIEW

**Comment by ingkebil** [comment]
Rebased here as well.

File: cg/store/api/status.py [hunk 1]
```diff
@@ -408,9 +408,30 @@ def analyses_to_deliver(self):
...
+            .join(models.Family, models.Family.links, models.FamilySample.sample)
+            .filter(
+                models.Sample.delivered_at.isnot(None),
+                or_(
+                    models.Analysis.delivery_report_created_at.is_(None),
+                    and_(
+                        models.Analysis.delivery_report_created_at.isnot(None),
+                        models.Analysis.delivery_report_created_at < models.Sample.delivered_at
+                    )
+                )
```
Comment (ingkebil) [inline cg/store/api/status.py]: here's an awesome request: a unit test for this query :D

Comment (patrikgrenfeldt) [reply cg/store/api/status.py]: done!

**Comment by barrystokman** [comment]
Tested and approved.

**Pull request merged**